TweetBook/Program.cs

```diff
@@ -1,7 +1,11 @@
+﻿using TweetBook.Services;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 
+builder.Services.AddSingleton<IPostService, PostService>();
+
 builder.Services.AddControllers();
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
```

Comment:
Maybe you can delete all the non-relevant to the project comments like this one. Besides this, you can keep the ones which help you understand your code, although in real life we put comments only in exceptional situations where not any developer would understand at first glance the code.